pyfar/dsp/fft.py

```diff
@@ -105,11 +105,109 @@ def irfft(spec, n_samples, sampling_rate, fft_norm):
     spec = normalization(spec, n_samples, sampling_rate, fft_norm,
                          inverse=True, single_sided=True)
     # Inverse DFT
-    data = fft.irfft(
+    data = _fft.irfft(
         spec, n=n_samples, axis=-1, workers=multiprocessing.cpu_count())
 
     return data
 
+def fftfreq(n_samples, sampling_rate):
```

Comment:
Functions must be separated by 2 blank lines. Otherwise flake8 will raise an error.

Reply:
* everything should be formatted according to flake8